features/checkout/seeing_order_summary/seeing_current_prices_of_products_after_catalog_promotion_becomes_ineligible.feature

```diff
@@ -0,0 +1,23 @@
+@checkout
+Feature: Seeing current prices of products after catalog promotion becomes ineligible
+    In order to buy products in its correct prices
+    As a Customer
+    I want to have products with its current prices in the cart
```

Comment:
`I want to have products with their original prices in the cart`?

Reply:
the current sentence has more sense for me because in this scenarios I want to have a valid and current price, for customer, it is irrelevant if it is original or discounted (by different catalog promotion for example)

Reply:
I was referring that the price that Customer sees is *always* current for they :) but it's not relevant for know, I would only change `its` to `their` 🖖